docs/conf.py

```diff
@@ -120,6 +121,12 @@
 # so a file named "default.css" will overwrite the builtin "default.css".
 html_static_path = ['_static']
 
+
+# Add any extra paths that contain custom files (such as robots.txt or
+# .htaccess) here, relative to this directory. These files are copied
+# directly to the root of the documentation.
+html_extra_path = ['']
```

Comment:
Can you tell me what you are trying to achieve here? The above reveals the whole source tree, including `conf.py` in the published HTML tree, which for various reasons is wrong.

![image](https://user-images.githubusercontent.com/374612/74539542-7f44bf80-4f3e-11ea-945e-fef49906730f.png)

Reply:
Corrected, thank you!